cachito/workers/pkg_managers/gomod.py

```diff
@@ -239,9 +243,11 @@ def resolve_gomod(app_source_path, request, dep_replacements=None, git_dir_path=
             new_name = dep_replacement.get("new_name", name)
             version = dep_replacement["version"]
             log.info("Applying the gomod replacement %s => %s@%s", name, new_name, version)
-            run_gomod_cmd(
-                ("go", "mod", "edit", "-replace", f"{name}={new_name}@{version}"), run_params
-            )
+            with tracer.start_as_current_span("run gomod " + new_name):
```

Comment:
Probably not worth having a specific trace for this piece here. This is for dependency replacements defined in the Cachito request itself, and AFAIK, nobody is using it.